test/integration/proposals_config.ts

```diff
@@ -19,6 +20,23 @@ const proposals: ProposalsConfigMap = {
     category: ProposalCategory.None,
     totalValue: 0,
     proposal: undefined
+  },
+  fip_62: {
+    deploy: true,
+    proposalId: undefined,
+    affectedContractSignoff: [
+      'ethPSM',
```

Comment:
add ethPSM to PCVGuardian map

Reply:
added ethPSM to PCVGuardian and we are using the correct eth price oracle.